app/models/attendee.js

```diff
@@ -34,19 +34,13 @@ export default ModelBase.extend({
   gender             : attr('string'),
   ageGroup           : attr('string'),
   birthDate          : attr('moment'),
-  complexFieldValues : attr(),
+  complexFieldValues : attr({ defaultValue: () => {} }),
```

Comment:
That's default value set to an empty function

Reply:
but it will return an empty object

Reply:
No